### ISSUE TITLE
No way to set NODE_ENV env var

### ISSUE THREAD

**Issue by adsteel** [SublimeLinter/SublimeLinter-eslint#227]
We have a check for NODE_ENV in our webpack 1 config. If it is not set, we console log out a warning that it is being set to default. Because it is not set for the linter, this warning pops open a window in ST every time I make a change, console logging out that message. Very distracting. It would be nice to be able to set that variable for the linter, or (even better) import the ENV variables defined in my terminal session.

**Comment by kaste** [comment]
If you log out something, you should log to stderr not stdout. stdout should only contain the eslint output. 

sublime will inherit the env only if you start it from the terminal. But you can tweak the execution env. See last paragraph in https://github.com/SublimeLinter/SublimeLinter-eslint/blob/master/messages/4.0.1.txt

**Comment by adsteel** [comment]
It doesn't seem to inherit from the terminal. Even when I declare the var during startup, like so (or without).

```bash
# quit sublime text
$ echo $NODE_ENV
development
$ subl . # fails
$ NODE_ENV=development subl . # also fails
```

Adding `NODE_ENV` to the `env` object in the eslintrc file raises an error of unknown environment.

**Comment by kaste** [comment]
The env object goes not into eslintrc but the SublimeLinter settings (global or per project). 

The other thing confuses me bc it for sure inherits the PATH, no?

**Comment by adsteel** [comment]
I did a `rm -rf node_modules && npm i` and now the problem is gone. I have no idea what was happening. Thanks for the quick response @kaste!

**Comment by kaste** [comment]
Regularly rm -rf for the win 🤹
